config/fund_loader_config/cof/cof_r2.py

```diff
@@ -38,6 +40,47 @@
                 "community-ownership-fund-privacy-notice"
             ),
         ),
+        "reference_contact_page_over_email": True,
+        "contact_us_banner_json": {
+            "en": textwrap.dedent(
+                """
+                 <h2 class="govuk-heading-m">Get application support</h2>
+                 <p class="govuk-body">
+                    <a class="govuk-link" href="https://mycommunity.org.uk/community-ownership-fund">Visit the My Community website</a>
+                    for information and guidance on applying to Community Ownership Fund.
+                    <a class="govuk-link" href="https://mycommunity.org.uk/community-ownership-fund#enquiry-form">Fill out the enquiry form</a>
+                    to request advice from My Community.
+                </p>
+                <p class="govuk-body">
+                    We cannot provide direct support to applicants outside of this service.
+                </p>
+                <h2 class="govuk-heading-m">Get technical support</h2>
+                <p  class="govuk-body">
+                    Contact the Department of Levelling Up, Housing and Communities funding team if you need
+                    help with accessing or submitting an application form.
+                </p>
+            """
+            ),
+            "cy": textwrap.dedent(
+                """
+                 <h2 class="govuk-heading-m">Get application support</h2>
+                 <p class="govuk-body">
+                    <a class="govuk-link" href="https://mycommunity.org.uk/community-ownership-fund">Visit the My Community website</a>
+                    for information and guidance on applying to Community Ownership Fund.
+                    <a class="govuk-link" href="https://mycommunity.org.uk/community-ownership-fund#enquiry-form">Fill out the enquiry form</a>
+                    to request advice from My Community.
+                </p>
+                <p class="govuk-body">
+                    We cannot provide direct support to applicants outside of this service.
+                </p>
+                <h2 class="govuk-heading-m">Get technical support</h2>
+                <p  class="govuk-body">
+                    Contact the Department of Levelling Up, Housing and Communities funding team if you need
+                    help with accessing or submitting an application form.
+                </p>
+            """
+            ),
+        },
```

Comment:
Have we got a ticket for the welsh translations?

Reply:
We do yeah